docs/drafts/badges.rst

```diff
@@ -0,0 +1,144 @@
+Badges
+======
+
+This is a summary of the "badge concept" user story. The concepts are
+not yet implemented in Adhocracy 3.
+
+Adhocracy 2 knows a variety of badges and similar entities. The aim is
+to streamline these entities into more consistent or possibly independent
+concepts in Adhocracy 3. Some aspects shall not be implemented in A3.
+
+This document can be replaced once the concepts are ready in A3.
+
+
+Badge features in Adhocracy 2
+-----------------------------
+
+The various badges, categories and tags can be categorized as the
```

Comment:
This is defined briefly in the section *Badges, categories and tags in A2* below.